extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/templates/ingress.yaml

```diff
@@ -20,7 +19,7 @@ spec:
             pathType: {{ .pathType }}
             backend:
               service:
-                name: {{ .backend.service.name | default $releasename }}
+                name: {{ .backend.service.name }}
```

Comment:
S: Concerns here for projects upgrading from 1.9. They'll probably _not_ have service.name set since we were previously defaulting it.

Reply:
Cheap option might just be to use the `require` function and provide a helpful message. https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions